packages/react/src/core/utils/findBestMatch.test.ts

```diff
@@ -0,0 +1,40 @@
+import { findBestMatch } from './findBestMatch';
+
+const testLinks = [
+	{ href: '/' },
+	{ href: '/about' },
+	{ href: '/services' },
+	{ href: '/contact' },
+];
+
+describe('findBestMatch', () => {
```

Comment:
@nathsimpson @dagoof tests for the `findBestMatch` utility function can be found here. This function has previously been untested.